templates/reviewer/review/reviewerRecommendations.tpl

```diff
@@ -25,7 +25,7 @@
 		required=$required|default:true
 		disabled=$readOnly
 		translate=false
-		defaultValue=""
+		defaultValue=" "
```

Comment:
This approach will skip front-end input validation altogether but we need to skip that only for save for later button.

Looks like associated JS handler can be adjusted: https://github.com/pkp/pkp-lib/blob/e6cfd21cc0bbbcc82d95bdab8121bab8854fe439/js/controllers/form/reviewer/ReviewerReviewStep3FormHandler.js#L69
At the first glance I don't see an element with ID `recommendation` but there is `reviewerRecommendationId`. And this approach deals with required attribute depending on which button is pressed.

Reply:
Thanks for pointing to the right direction .